examples/voice_create_webhook.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import argparse
+import messagebird
+from messagebird.voice_webhook import VoiceCreateWebhookRequest
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--accessKey', help='access key for MessageBird API', type=str, required=True)
+parser.add_argument('--url', help='url for the webhook', type=str, required=True)
+parser.add_argument('--title', help='title for the webhook', type=str)
+parser.add_argument('--token', help='token for the webhook', type=str)
+args = vars(parser.parse_args())
+
+try:
+    client = messagebird.Client(args['accessKey'])
+
+    create_webhook_request = VoiceCreateWebhookRequest(url=args['url'], title=args['title'], token=args['token'])
+    webhook = client.voice_create_webhook(create_webhook_request)
+
+    # Print the object information.
+    print('\nThe following information was returned as a Voice Webhook object:\n')
+    print('  id                 : %s' % webhook.id)
+    print('  token              : %s' % webhook.token)
+    print('  url                : %s' % webhook.url)
+    print('  createdAtDatetime  : %s' % webhook.createdDatetime)
+    print('  updatedAtDatetime  : %s' % webhook.updatedDatetime)
+
+except messagebird.client.ErrorException as e:
```

Comment:
`ErrorException`, `Error` or `Exception`

Reply:
If it is already the case, just leave it haha

Reply:
Unfortunately, it was here already :)